templates/testutil/fs.go

```diff
@@ -102,7 +104,18 @@ func LoadDirMode(tb testing.TB, dir string) map[string]ModeAndContents {
 		if err != nil {
 			return err
 		}
+		relPath, err := filepath.Rel(dir, path)
+		if err != nil {
+			tb.Fatal(err)
+		}
+		skip := skippable(tb, relPath, opt.skipGlobs)
 		if d.IsDir() {
+			if skip {
```

Comment:
This part looks a little bit confusing to me? Can we `if skip` as the outlayer and in the inner layer, check whether `d.IsDir()`.

Reply:
Yeah, that is better, thanks. Done.